heron/storm/src/java/shade.conf

```diff
@@ -0,0 +1,4 @@
+rule com.google.** org.apache.storm.shaded.@0
+rule com.twitter.** org.apache.storm.shaded.@0
```

Comment:
hmm, `com.twitter` is different from 3rdparty dependency, these are all classes from our own project. Not shading them would be better I suppose.